db.scheme

```diff
@@ -5,7 +5,8 @@ CREATE TABLE IF NOT EXISTS `person` (
 
 CREATE TABLE IF NOT EXISTS `restaurant` (
  `id`   INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
- `name` TEXT NOT NULL
+ `name` TEXT NOT NULL,
+ `comment` TEXT NOT NULL
```

Comment:
Not sure why you'd want to add a default value though, let the user supply a default empty string if he so wishes.
Existing DB needs to be upgraded, and since we don't have any kind of versioning we can't do that automatically. Are we really at the point where backwards compatibility is a thing?